Automattic/course
# Add single post template

Implements https://github.com/Automattic/sensei-theme/issues/12

## Changes proposed

Added pattern for single post

## Testing instructions

- Create a new post, make sure the default template for post is selected
- Add a featured image to it
- Add some contents to the post
- Save and check the post from frontend
- Make sure it presents your post content in the same structure as the design

## Screenshot

<img width="393" alt="Screen Shot 2022-11-03 at 12 09 27 AM" src="https://user-images.githubusercontent.com/6820724/199581327-393ce3fe-b211-4f05-97dd-a472fe7da1e8.png">
